package.json

```diff
@@ -1506,6 +1506,16 @@
           "default": 500,
           "markdownDescription": "Defines the delay in milliseconds for chktex to wait after stopped typing.\nThis config only matters when `latex-workshop.chktex.run` is set to `onType`."
         },
+        "latex-workshop.chktex.convertOutput.column.enabled": {
+          "type": "boolean",
+          "default": true,
+          "markdownDescription": "Enable converting ChkTeX outputs' column numbers for non-ASCII characters."
+        },
+        "latex-workshop.chktex.convertOutput.column.chktexrcTabSize": {
+          "type": "string",
+          "default": "auto",
+          "markdownDescription": "Write the `TabSize` number in `.chktexrc`. If the value is `auto`, LaTeX Workshop will try to read the value from `.chktexrc`. The file in the home directory will not be read."
```

Comment:
It took me a while to understand the description. Maybe something like "Write the value of `TabSize` from `.chktexrc`" would be clearer. 

Why not reading the gobal `$HOME/.chktexrc` at least on Unix systems? Of course, https://github.com/James-Yu/LaTeX-Workshop/blob/61b3bbe1522428638baba7a81dfaf403f31c53b1/src/components/linter.ts#L18 would need to be updated.

Reply:
I have edited as suggested. For paths to search, I referred to the subsection "6.1.3 The chktexrc file" in `texdoc chktex`.